app/views/how-tos/index.html

```diff
@@ -20,6 +20,40 @@ <h1>How to guides</h1>
 
       <p class="nhsuk-lede-text">These guides will show you how to use the prototype kit, from creating a simple page to building complex user journeys.</p>
 
+      <h2>Get started</h2>
+      <ul class="govuk-list govuk-list--bullet">
```

Comment:
The rest of the lists on this page don’t use bullets, so match that? Also wrong prefix 😉 

```suggestion
      <ul class="nhsuk-list">
```